README.md

```diff
@@ -1,70 +1,137 @@
+a release payload image for a newer OpenShift version, CVO reconciles all
+Cluster Operators to their updated versions, and in turn, the Cluster Operators
+similarly update their operands.
+
+## OpenShift Upgrades
+
+For information about upgrading OpenShift clusters, please see the respective
+documentation:
+
+- [OKD Documentation: Updating clusters](https://docs.okd.io/latest/updating/index.html)
+- [Red Hat OpenShift Container Platform: Updating clusters](https://docs.openshift.com/container-platform/latest/updating/index.html)
+
+## `ClusterVersion` Resource
+
+Like other Cluster Operators, the Cluster Version Operator is configured by
+a Config API resource in the cluster: a [`ClusterVersion`](https://docs.openshift.com/container-platform/latest/rest_api/config_apis/clusterversion-config-openshift-io-v1.html):
+
+```console
+$ oc explain clusterversion
+  KIND:     ClusterVersion
+  VERSION:  config.openshift.io/v1
+
+  DESCRIPTION:
+       ClusterVersion is the configuration for the ClusterVersionOperator. This is
+       where parameters related to automatic updates can be set. Compatibility
+       level 1: Stable within a major release for a minimum of 12 months or 3
+       minor releases (whichever is longer).
+
+  FIELDS:
+     ...
+     spec	<Object> -required-
+       spec is the desired state of the cluster version - the operator will work
+       to ensure that the desired version is applied to the cluster.
+
+     status	<Object>
+       status contains information about the available updates and any in-progress
+       updates.
 ```
 
-1. This builds image locally and then pushes `${VERSION}` and `latest` tags to `${REPO}/origin-cluster-version-operator`.
-
-2. `${VERSION}` encodes the Git commit used to build the images.
-
-## Building release image using local CVO
+`ClusterVersion` resource follows the established [Kubernetes pattern](https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/#object-spec-and-status) where
+the `spec` sub-resource describes the desired state that CVO should achieve and
+maintain, and the `status` sub-resource that describes the state observed by CVO
+and its current status.
 
-1. Make sure you have `oc` binary from https://github.com/openshift/origin master as it requires `adm release` subcommand.
+In a typical OpenShift cluster, there will be a cluster-scoped `ClusterVersion`
+resource called `version`:
 
-2. Run the following command to create release-image at `docker.io/abhinavdahiya/origin-release:latest`:
-
-```sh
-oc adm release new -n origin \
-    --from-image-stream=origin-v4.0 \
-    --to-image-base=docker.io/abhinavdahiya/origin-cluster-version-operator:latest \
-    --to-image docker.io/abhinavdahiya/origin-release:latest
+```console
+$ oc get clusterversion version
+NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
+version   4.11.3    True        False         85d     Cluster version is 4.11.3
 ```
```

Comment:
Around here it is probably worth pointing out that `oc adm upgrade` is a nicer way to get pretty-printed ClusterVersion information, without `get`'s default output format's attempt to pack all of the information into a single line.